templates/services/common/cf/taskrole.yml

```diff
@@ -52,14 +52,14 @@ TaskRole:
               Resource: '*'
               Condition:
                 StringEquals:
-                  'aws:ResourceTag/ecs-project': !Sub '${ProjectName}'
+                  'aws:ResourceTag/ecs-project': !Sub '${AppName}'
```

Comment:
```suggestion
                  'aws:ResourceTag/copilot-project': !Sub '${AppName}'
```

Reply:
that's on purpose, since this should belong to `deploy` package I think.

Reply:
oh nvm i should make those changes. thanks!